### DISCUSSION TITLE
Ravendb 3.5 with file system

### DISCUSSION THREAD

**Discussion by RDF-FEI** [ravendb/ravendb#17111] [General]
We are still using the old Ravendb3.5 File system for application attachment storage.  We create a primary and secondary databases and sync them using the the Ravendb sync. Recently, we found a network problem and discover the sync was stuck for a month. After fixing the network problem, the Ravendb is no longer syncing the data over to the secondary server. Is there a way to force it to sync without rebuild this secondary db from scratch?

**Comment by RDF-FEI** [comment]
Anyone? Appreciated if someone can help.

**Comment by ayende** [comment]
FYI, I assume you know, but RavenDB 3.5 is no longer supported, unless you have a legacy support contract with us.
If you have such a contract, you'll need to ping the support channel directly. In such a case, please provide full logs from both servers.

Something to note is whatever there is _any_ sync going on between the servers, or if it just stopped.
